src/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <title>Meteo Schweiz Opendata Explorer</title>
-    <base href="/" />
+    <base href="./" />
```

Comment:
I'm curious: Why is the `.` necessary?

Reply:
The angular application is not served on `/` of the url but on `/224488-MeteoSchweiz-WebGUI/`. So when assets are loaded using a base of `/` the browser tries to load them from `/<asset>`. With the relative base the assets are loaded relative to the loaded html file. I just noticed that this will not work when loading the page with nested routes, so I'll have to change this again.

This was not apparent for gb3 as it is deployed to the root of a custom domain name.

Reply:
Ah. That makes a lot of sense, thanks for the explanation ❤
This issue could probably be solved using the `"baseHref": "224488-MeteoSchweiz-WebGUI"` setting within `angular.json`.